### PR TITLE
[FIX] tools: test pot with nondeterminism

### PR DESCRIPTION
**Pull request by HydrionBurst**
The test added in 09a4762b will fail randomly, because of the nondeterministic
os.walk access. This commit allows this nondeterminism in the test.

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: odoo/addons/test_translation_import/tests/test_term_count.py [hunk 1]
```diff
@@ -245,6 +245,7 @@ def test_export_pot(self):
         with io.BytesIO(base64.b64decode(pot_file_data)) as pot_file:
             pot_file.name = f'{module_name}.pot'
             for line1, line2 in zip(TranslationFileReader(pot_file, 'po'), TranslationFileReader(file_path(f'{module_name}/i18n/{module_name}.pot'), 'po')):
+                self.assertEqual(set(line1.pop('comments').split('\n')), set(line2.pop('comments').split('\n')))
```
Comment (rco-odoo) [inline odoo/addons/test_translation_import/tests/test_term_count.py:248]: You could use
```suggestion
                self.assertCountEqual(line1.pop('comments').split('\n'), line2.pop('comments').split('\n'))
```

File: odoo/tools/translate.py [hunk 2]
```diff
@@ -1294,6 +1294,7 @@ def _export_translatable_resources(self):
                     break
 
 
+
```
Comment (rco-odoo) [inline odoo/tools/translate.py:1297]: Oops :wink:

File: odoo/addons/test_translation_import/tests/test_term_count.py (hunk 1 above)
Comment (HydrionBurst) [reply odoo/addons/test_translation_import/tests/test_term_count.py:248]: Oh, I was misled by its name

**Review by rco-odoo — APPROVED** [review]
@robodoo delegate+

**Review by rco-odoo — APPROVED** [review]

**Comment by rco-odoo** [comment]
@robodoo delegate+

**Comment by rco-odoo** [comment]
@fw-bot ignore

**Comment by rco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**